python/paddle/nn/functional/loss.py

```diff
@@ -2229,3 +2230,124 @@ def hinge_embedding_loss(input, label, margin=1.0, reduction='mean', name=None):
         return paddle.sum(loss, name=name)
     elif reduction == 'none':
         return loss
+
+
+def triplet_margin_loss(input,
+                        positive,
+                        negative,
+                        swap=False,
+                        p=2,
+                        eps=1e-6,
```

Comment:
"eps" should be "epsilon" consistance with [RFC](https://github.com/PaddlePaddle/community/pull/31) and like other APIs.